server/spec/consumer_resource_spec.rb

```diff
@@ -671,6 +671,24 @@ def verify_installed_pids(consumer, product_ids)
 
   end
 
+  it 'should not list service levels from expired pools' do
+    product = create_product(random_string('product'),
+                              random_string('product'),
+                              {:attributes => {:support_level => 'Expired'},
+                               :owner => @owner1['key']})
+    create_pool_and_subscription(@owner1['key'], product.id,
+                                 1, [], '', '', '', Date.today, Date.today - 1)
```

Comment:
You have a range from today though yesterday. Let's use something valid like Date.today-2, Date.today-1.